client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,297 @@
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        thursday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        friday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        saturday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        sunday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+      };
+    },
+    {
+      setSubmitting,
+    }: FormikHelpers<{
+      name: string;
+      days: {
+        monday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        tuesday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        wednesday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        thursday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        friday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        saturday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+        sunday: {
+          active: boolean;
+          startTime: string;
+          endTime: string;
+        };
+      };
+    }>,
+  ) => {
+    CreateSchedule(values).then((data) => {
+      setDropDownSelected(false);
+      if (data.error) {
+        setSubmitting(false);
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        console.error({ data });
+        setSubmitting(false);
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
+  return (
+    <Box>
+      <SettingHeader header="Your Availability" />
+      <FormControl sx={{ width: '20%', height: '10%' }}>
+        <InputLabel id="scheduleName">Select</InputLabel>
+        <Select
+          id="scheduleName"
+          sx={{ marginRight: '1rem', fontWeight: 700 }}
+          name="schedule"
+          value={selectedSchedule}
+          onChange={(e) => handleChange(e)}
+        >
+          {schedules.map((schedule) => generateSchedules(schedule.name, schedule.id))}
+        </Select>
+      </FormControl>
+      <Button
+        variant="contained"
+        color="primary"
+        sx={{ height: '4em' }}
+        disableElevation
+        onClick={() => {
+          setNewSchedule(true);
+          setDropDownSelected(false);
+        }}
+      >
+        + New Schedule
+      </Button>
+      {newSchedule && (
+        <Formik
+          initialValues={{
+            name: '',
+            days: {
+              monday: {
+                active: false,
```

Comment:
Since the initial values are identical, you can store them in an object and reuse the object here

Reply:
Done the changes